src/process/child.rs

```diff
@@ -63,10 +86,16 @@ impl ChildProcess {
             .as_mut()
             .expect("Complete the setup of uds in advance.");
 
-        let mut events = Events::with_capacity(128);
-        poll.poll(&mut events, Some(Duration::from_millis(1000)))?;
+        // Create collection with capacity to store up to 128 events
+        let mut events = Events::with_capacity(MAX_EVENTS);
+        // poll the receiving end of pipe created for 1 seconds for an event
```

Comment:
It is the same as this.
https://github.com/containers/youki/pull/64/files#r646003233